rpgs/views.py

```diff
@@ -76,12 +82,14 @@ def add_to(request):
 	# Also worth noting that this kind of form is exactly the kind of thing
 	# that could be enhanced with AJAX, and that could shape the response structure.
 	rpg = get_object_or_404(Rpg, id=request.POST.get('id'))
+	url = reverse('rpg', kwargs={'pk':request.POST.get('id')})
 	if request.user.member == rpg.creator:
 		users = User.objects.filter(username=request.POST.get('username'))
 		if users.count() == 0:
 			return HttpResponseRedirect(reverse('rpg', kwargs={'pk':request.POST.get('id')}))
 		rpg.members.add(users[0].id)
-	return HttpResponseRedirect(reverse('rpg', kwargs={'pk':request.POST.get('id')}))
+		notify(users[0].member, 'rpg_added', 'You were added to "'+rpg.title+'".', url)
```

Comment:
Is there a better way to find the creator than `users[0]`?

Reply:
Like, we have `rpg.creator`, right?

Reply:
Turns out `users[0]` was something I wrote, and we can worry about it another time